src/ploomber/telemetry/telemetry.py

```diff
@@ -311,6 +296,84 @@ def check_first_time_usage():
     return not uid_path.exists() and config_path.exists()
 
 
+def get_latest_version():
+    """
+    The function checks for the latest available ploomber version
+    uid file doesn't exist.
+    """
+    conn = httplib.HTTPSConnection('pypi.org', timeout=1)
+    try:
+        conn.request("GET", "/pypi/ploomber/json")
+        content = conn.getresponse().read()
+        data = json.loads(content)
+        latest = data['info']['version']
+        return latest
+    except Exception:
+        return __version__
+    finally:
+        conn.close()
+
+
+def read_conf_file(conf_path):
+    try:
+        with conf_path.open("r") as file:
+            conf = yaml.safe_load(file)
+    except Exception as e:
+        warnings.warn(f"Error: Can't read config file {e}")
+    return conf
+
+
+def write_conf_file(conf_path, to_write, error=None):
+    try:  # Create for future runs
+        with conf_path.open("w") as file:
+            yaml.dump(to_write, file)
+    except Exception as e:
+        warnings.warn(f"ERROR: Can't write to config file: {e}")
+        if error:
+            return e
+
+
+def check_version():
+    """
+    The function checks if the user runs the latest version
+    This check will be skipped if the version_check_enabled is set to False
+    If it's not the latest, notifies the user and saves the metadata to conf
+    Alerting every 2 days on stale versions
+    """
+    # Read conf file
+    config_path = Path(check_dir_exist(CONF_DIR), 'config.yaml')
+    conf = read_conf_file(config_path)
+    # Check if the flag was disabled
+    if conf and 'version_check_enabled' in conf.keys() \
+            and not conf['version_check_enabled']:
+        return
+
+    # If latest version, do nothing
+    latest = get_latest_version()
+    version = __version__
+    print(version)
+    if __version__ == latest:
+        return
+
+    today = datetime.datetime.now()
+
+    # First time the version is checked
+    if 'version_check_enabled' in conf.keys():
+
+        # Check if we already notified in the last 2 days
+        last_message = conf['version_check_enabled']
+        diff = (today - last_message).days
+        if diff < 2:
+            return
+
+    click.echo("****Please update the ploomber version, run:\n"
```

Comment:
this `***` at the beginning look odd 😂 maybe remove them. also, click has a `secho` function that allows you to change the font color, i think it'd be good to show it in yellow to draw more attention to it